chart/openfaas/templates/alertmanager-dep.yaml

```diff
@@ -27,14 +27,24 @@ spec:
           - "--config.file=/alertmanager.yml"
           - "--storage.path=/alertmanager"
         livenessProbe:
-          httpGet:
-            path: /#/status
-            port: 9093
+          exec:
```

Comment:
We should have a note somewhere in the helm chart or README of the repo explaining why we use this health-check instead of `httpGet`. Please add this before the merge.

Reply:
Done